### DISCUSSION TITLE
Fresh, merged language file from Crowdin

### DISCUSSION THREAD

**Discussion by kzsa** [directus/directus#7741] [Archive]
Hi,
🙏🙏🙏 Does anyone have a fresh merged en-US.yaml en-HU.yaml file from Crowdin?

Reasons:
1. faster translation
2. I could check the context in a simple way

Crowdin is a very fancy translation method, but it didn't work for me. I find the workflow too slow. 🤷‍♂️

Thx: kzsa

**Comment by rijkvanzanten** [comment]
Heyo! They're all in here:

https://github.com/directus/directus/tree/main/app/src/lang/translations

I also believe crowdin has a way to download/upload this file, so it'll still go through the regular flow 👍🏻

**Reply by kzsa** [reply]
Hi,
thank you very much for your quick reply. I have already looked at these. They only include content that has already been translated and I didn't want to mess around with merging nested objects, then checking everything is in place. If I have time I will have a look. Crowdin options: translate and vote.

Br: kzsa

**Reply by rijkvanzanten** [reply]
Fair! I think you might be able to rely on Crowdin's export/import functionality then 🙂 

<img width="781" alt="CleanShot 2021-09-01 at 09 54 54@2x" src="https://user-images.githubusercontent.com/9141017/131684143-88c85249-8a8b-4866-9b33-647298b15a47.png">


I believe that should take care of finding duplicates, merging etc etc

**Reply by kzsa** [reply]
Thank you very much!
